src/reports/components/instance-details.scss

```diff
@@ -51,80 +51,66 @@
     @include fill-available-width;
 
     border-radius: inherit;
-
-    th {
-        padding: 12px 20px;
-
-        vertical-align: top;
-    }
-
-    .row {
-        top: calc(50% - 20px / 2);
-        border-bottom: 0.5px solid $neutral-10;
-    }
-
-    .row:last-child {
-        border-bottom: none;
-    }
-}
-
-.report-instance-table .label {
-    font-size: 14px;
-    line-height: 20px;
-    font-family: $semiBoldFontFamily;
-    color: $primary-text;
-    text-align: left;
-    width: 70px;
-}
-
-.report-instance-table .instance-list-row-content {
-    color: $secondary-text;
-    padding: 12px 20px;
-    font-size: 14px;
-    line-height: 20px;
-    align-items: flex-end;
-    display: flex;
-}
-
-.report-instance-table .instance-list-row-content.content-snipppet {
-    font-family: Consolas, Segoe UI, Courier New, monospace;
-    white-space: normal;
-    word-wrap: break-word;
-    word-break: break-word;
-}
-
-table.report-instance-table {
     border-collapse: collapse;
-    overflow-x: auto;
-    word-break: break-word;
-}
 
-@media screen and (max-width: 480px) {
-    .report-instance-table tr {
-        display: block;
-    }
-
-    .report-instance-table td {
-        display: block;
+    overflow-wrap: break-word;
+    word-break: break-word;
 
-        .label {
-            text-align: left;
+    .row {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+
+        // The padding on the parent and margins on the children are coupled; they
+        // are intended to produce the desired spacing regardless of whether the children
+        // are presented side-by-side or wrapped
+        padding-top: 2px;
+        padding-bottom: 14px;
+        padding-left: 0px;
```

Comment:
I tried removing this line and it had no effect on the styling. We might be able to get rid of this.

Reply:
I agree that it isn't currently necessary; I included it (and the 0 margins a few lines down) because I found it more readable to be explicit (there are some elements that browsers will use a 1px padding/margin for by default, and this saves a reader/reviewer from having to think about that/look up whether it applies)